test/types/chplhashtable/nonPodRecord.chpl

```diff
@@ -0,0 +1,32 @@
+use Set;
```

Comment:
It'd be nice to rename this and the other test something like "nonTrivialEqRecord.chpl"

Reply:
Good thought, done.  I also added a test for the case you suggested, which I had also been wondering about.  I think the error/behavior could be better, but at least it doesn't hash for now...